pkgs/faf-client/default.nix

```
@@ -0,0 +1,71 @@
+{
+  lib,
+  stdenv,
+  makeWrapper,
+  makeDesktopItem,
+  # regular jdk doesnt work due to problems with JavaFX even with .override { enableJavaFX = true; }
+  openjdk17-bootstrap,
+  xorg,
+  libGL,
+  gtk3,
+  glib,
+  alsa-lib,
+}: let
+  pname = "faf-client";
+  version = "2023.1.2";
+  src = builtins.fetchTarball {
+    url = "https://github.com/FAForever/downlords-faf-client/releases/download/v${version}/faf_unix_${builtins.replaceStrings ["."] ["_"] version}.tar.gz";
+    sha256 = "0lfyfbvjzfgkq3b485d8ma8fjc3hbc5p5h2ffl3lrvai6ksa59g2";
+  };
+  meta = with lib; {
+    description = "Official client for Forged Alliance Forever";
+    homepage = "https://github.com/FAForever/downlords-faf-client";
+    license = licenses.mit;
+  };
+  icon = builtins.fetchurl {
+    url = "https://faforever.com/images/faf-logo.png";
+    name = "faf-client.png";
+    sha256 = "0z8sxbmq50zwi5vb8b2hyhhg5809bnimrzz6l57c5vw9ia038a5v";
+  };
+  desktopItem = makeDesktopItem {
+    inherit icon;
+    name = "faf-client";
+    exec = "faf-client";
+    comment = meta.description;
+    desktopName = "Forged Alliance Forever";
+    categories = ["Game"];
+    keywords = ["FAF" "Supreme Commander"];
+  };
+  libs = [
+    xorg.libXxf86vm
+    libGL
+    gtk3.out
+    glib
+    alsa-lib
+  ];
+in
+  stdenv.mkDerivation {
```

```suggestion
  stdenvNoCC.mkDerivation {
```
If we're not using the C compiler, we might as well use this.